Library/Config.ts

```diff
@@ -92,6 +101,16 @@ class Config {
             "*.core.eaglex.ic.gov"
         ];
 
+        this.isAuthRequired = csCode.authorization && csCode.authorization === "aad";
```

Comment:
There was a discussion on the Spec doc.  suggestion is to use AAD instead of aad.  Unless the connectionstring parser tolowers everything you need to update this.